### PR TITLE
Added first files and rules

### PR DESCRIPTION
**Pull request by qmg-mrivelles**
We're going to start including the repo in Utonium, and start to enable rules one by one to not do a massive refactor.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+.idea/
+.DS_Store
```
Comment (iperezqm) [inline .gitignore]: please add trailing newline

File: index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+module.exports = {
+  extends: [
```
Comment (iperezqm) [inline index.js]: ``extends: require.resolve('eslint-config-airbnb')``. No need to map over an array with a single element

File: index.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    "import/no-extraneous-dependencies": [ "error", { "devDependencies": true, "optionalDependencies": false, "peerDependencies": false } ],
+    "react/require-extension": "off",
+    "new-cap": "off",
+    "react/jsx-space-before-closing": "off",
+    "react/jsx-tag-spacing": [ "error", { "beforeSelfClosing": "always" } ],
+    "jsx-a11y/aria-role": "off",
+    "react/jsx-wrap-multilines": "off",
+    "no-else-return": "off",
+
+    // Rules we torn off to pass PR and Merge
```
Comment (iperezqm) [inline index.js]: torn => turned

Comment (iperezqm) [inline index.js]: also, there's rules on the list below that are superseding some of our pre-existing rules.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    "browser": true
+  },
+  globals: {
+    "arguments": true
+  },
+  ecmaFeatures: {
+    "jsx": true
+  },
+  rules: {
+    // Previous Rules
```
Comment (iperezqm) [inline index.js]: Are none of these already present in some form or another on the preset we are extending? We should avoid having them here if they are on the preset

File: package.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  "license": "Unlicense",
+  "bugs": {
+    "url": "https://github.com/qmetric/eslint-config-qmetric/issues"
+  },
+  "homepage": "https://github.com/qmetric/eslint-config-qmetric",
+  "dependencies": {
+    "eslint-config-airbnb": "^11.2.0"
+  },
+  "devDependencies": {
+    "eslint": "^3.19.0",
```
Comment (iperezqm) [inline package.json]: Please review ``devDependencies`` and ``peerDependencies`` Remove duplications and use both sections accordingly.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  "peerDependencies": {
+    "eslint": "^3.19.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^5.0.3",
+    "eslint-plugin-react": "^7.0.1",
+    "eslint-config-airbnb": "^15.0.1"
+  },
+  "engines": {
+    "node": ">= 4"
```
Comment (iperezqm) [inline package.json]: I'd even say ``>=6``

File: .gitignore (hunk 1 above)
Comment (qmg-mrivelles) [inline .gitignore]: Done!

File: index.js (hunk 2 above)
Comment (qmg-mrivelles) [inline index.js]: I was with an eye to keep it open in the case of we include other configs, but yes we can modify this later 💃

File: index.js (hunk 4 above)
Comment (qmg-mrivelles) [inline index.js]: Cleaned!

File: index.js (hunk 3 above)
Comment (qmg-mrivelles) [inline index.js]: Cleaned.

File: package.json (hunk 5 above)
Comment (qmg-mrivelles) [inline package.json]: Yes, only we need peerDependencies, as we do not plan to add tests here.

File: package.json (hunk 6 above)
Comment (qmg-mrivelles) [inline package.json]: done!

File: package.json [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "config",
+    "qmetric",
+    "javascript",
+    "styleguide"
+  ],
+  "author": "Marc Rivelles (https://twitter.com/@marchriv)",
```
Comment (iperezqm) [inline package.json]: We use QMetric as the author in our packages

File: index.js [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    "flowtype/space-before-type-colon": [ 2, "never" ],
+    "flowtype/space-after-type-colon": [ 2, "always" ],
+
+    // Rules that we don't want
+    "react/jsx-key": "off",
+    "object-curly-spacing": "off",
+    "no-plusplus": "off",
+    "max-len": "off",
+    "import/imports-first": "off",
+    // when we fix errors that is the configuration to avoid a deprecated rule "import/no-extraneous-dependencies": [ "error", { "devDependencies": true, "optionalDependencies": false, "peerDependencies": false } ],
```
Comment (iperezqm) [inline index.js]: turn them off

File: package.json (hunk 7 above)
Comment (qmg-mrivelles) [inline package.json]: Done

File: index.js (hunk 8 above)
Comment (qmg-mrivelles) [inline index.js]: Done

**Comment by qmg-mrivelles** [comment]
Is ready for PR.

**Pull request merged**